tests/ci/style_check.py

```diff
@@ -17,7 +17,7 @@
 from ssh import SSHKey
 from stopwatch import Stopwatch
 
-NAME = "Style Check"
+NAME = JobNames.STYLE_CHECK
```

Comment:
[can ignore]: I don't think we need this NAME variables in scripts

Reply:
Yes, there's a conflict in PR, so I'll rebase it with removing `NAME` vars from every touched file.